flytekit/clis/sdk_in_container/package.py

```diff
@@ -40,7 +40,7 @@
     required=False,
     type=click.Path(exists=True, file_okay=False, readable=True, resolve_path=True, allow_dash=True),
```

Comment:
Do you have a minimal working example that shows how the `--source` argument of `pyflyte package` is supposed to be used? I'm asking because I ever only use the `--pkgs` arg and I don't know in which situation it makes sense to use the `--source` arg. I feel as part of this issue the difference should become clear from the help messages.

Reply:
@fg91  I have no idea about working example, will this help message works?

> help="Local filesystem path to the root of the package. Use --source when specifying the root directory of your workflows. Example: --source /path/to/workflows. Use --pkgs to specify package names directly.",

Reply:
I think it's useful but will let @fg91 comment on this one

Reply:
I unfortunately also don't have an idea how the `--source` tag is supposed to be used over the `--pkgs` arg.

My proposition would be to merge this PR as is (after [this](https://github.com/flyteorg/flytekit/pull/2877/files#r1838708695) tiny nit has been merged) but to leave the issue open and then ask in Flyte's `ask-the-community` channel whether anyone knows. If yes, we can make a second PR explaining the diff. If really no one knows or there isn't any use for the `--source` arg, we can create an issue to deprecate it.
What do you think about this?